src/app/multi-editor/multi-editor.component.ts

```diff
@@ -27,8 +28,8 @@ export class MultiEditorComponent implements OnInit {
   previewedActions: UserActions;
   errorMessage: string;
   successMessage: string;
-  // records that are disselected
-  checkedRecords: string[] = [];
+  // records that are different from the general selection rule
+  selectedRecordsMap: {[uuid: string]: boolean} = {};
```

Comment:
`recordSelectionStatus`. this component is getting too big so we can't tolerate misleading names.